Issue 2764 (#2804)
rotki/rotki: docs/changelog.rst

Changelog
=========

> * :feature:`2764` Users will now be able to restore the default assets to their initial state.

Reviewer: ```suggestion
* :feature:`2764` Users will now be able to restore the local assets database to its initial state.
```
